functions/exercise19/scriptSpect.js

```diff
@@ -0,0 +1,9 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals getElementLargerThanNumber*/
+
+describe("Returning array elements larger than a number", function () {
+    it("in the array [2, 5, 10, 15, 20, 30] elements larger than 10 are [15, 20, 30]", function () {
+        var elementLargerThanANumber = getElementLargerThanNumber(10,[2, 5, 10, 15, 20, 30]);
```

Comment:
file is not formatted - there should be a space beetween arguments as it is hard to read.